project/Build.scala

```diff
@@ -81,8 +81,8 @@ object Build {
    *  scala-library.
    */
   def stdlibVersion(implicit mode: Mode): String = mode match {
-    case NonBootstrapped => "2.13.1"
-    case Bootstrapped => "2.13.1"
+    case NonBootstrapped => "2.13.2-bin-db6fa85"
```

Comment:
```
2020-03-31T13:53:33.9581073Z [error] sbt.librarymanagement.ResolveException: Error downloading org.scala-lang:scala-library:2.13.2-bin-db6fa85
2020-03-31T13:53:33.9581395Z [error]   Not found
2020-03-31T13:53:33.9581618Z [error]   Not found
2020-03-31T13:53:33.9582243Z [error]   not found: /root/.ivy2/local/org.scala-lang/scala-library/2.13.2-bin-db6fa85/ivys/ivy.xml
2020-03-31T13:53:33.9582886Z [error]   not found: https://scala-webapps.epfl.ch/artifactory/sbt-plugin-releases/org.scala-lang/scala-library/2.13.2-bin-db6fa85/ivys/ivy.xml
2020-03-31T13:53:33.9583541Z [error]   not found: https://scala-webapps.epfl.ch/artifactory/central/org/scala-lang/scala-library/2.13.2-bin-db6fa85/scala-library-2.13.2-bin-db6fa85.pom
2020-03-31T13:53:33.9584678Z [error]   not found: http://repo.typesafe.com/typesafe/releases/org/scala-lang/scala-library/2.13.2-bin-db6fa85/scala-library-2.13.2-bin-db6fa85.pom
2020-03-31T13:53:33.9585329Z [error]   not found: https://oss.sonatype.org/content/repositories/snapshots/org/scala-lang/scala-library/2.13.2-bin-db6fa85/scala-library-2.13.2-bin-db6fa85.pom
2020-03-31T13:53:33.9585973Z [error]   not found: https://oss.sonatype.org/content/repositories/releases/org/scala-lang/scala-library/2.13.2-bin-db6fa85/scala-library-2.13.2-bin-db6fa85.pom
2020-03-31T13:53:33.9586676Z [error]   unauthorized: https://oss.sonatype.org/service/local/staging/deploy/maven2/org/scala-lang/scala-library/2.13.2-bin-db6fa85/scala-library-2.13.2-bin-db6fa85.pom (Sonatype Nexus Repository Manager)
```

I believe something is wrong with the resolution of this library. @martijnhoekstra, does it resolve locally when you try running the community build? Maybe some other steps need to be taken to make that snapshot version resolve?

Reply:
this is hosted at https://scala-ci.typesafe.com/artifactory/scala-integration

Reply:
Per this comment: https://github.com/scala/scala/pull/8282#issuecomment-594046566

Reply:
The resolver is added in our build below (`resolvers += ...`), but it's not added to the community-build projects (one would need to also set `resolvers` in https://github.com/lampepfl/dotty/blob/7a28eef2a65e769a4439aff67ff0d158db589003/community-build/test/scala/dotty/communitybuild/CommunityBuildTest.scala#L98)

Reply:
Maybe it's easiest to just wait until 13.2 lands.